### DISCUSSION TITLE
GO PRO for photoshooting

### DISCUSSION THREAD

**Discussion by Ed0ard02022** [gopro/OpenGoPro#231] [General]
Hi to all, I would like to «pair» or setup up to 8 (as array) GOPRO camera to be able to preview and shot concurrently
from 6 to 8 picture from different angles of the same product (0°, - 45°, 90°, 135°, 180°, 225°, 270°, 315°). 
I cannot connect through wifi  because I need the device (notebook or tablet) to be connected as I would implement  browser based interface. Therefore I’m  assuming I have as opportunity only:
 - usb 
 - bluetooth (but it cant’ show up preview!!)  
 
 or should I go for a lan connection for browser and wifi for GOPRO?  I i whish something could be developed through OPENGO pro api? 
 
 Any idea? 
 
 the interface would be a browser based solution for following features:
  I can wake up all GOPRO (BTE)
  I can preview all 8 GOPRO 
  I can recall specific GOPRO setting (ISO, shooting angle ...)
  I can "shoot"
  I can import from all 8 GOPRO the picture and place the shooting within a folder

**Comment by tcamise-gpsw** [comment]
You're going to need either USB or Wifi to view the preview (or webcam) and to download pictures. Since you say you can not use Wifi for the cameras, then I think your best bet is to use USB
